tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+coverage[toml]>=5.0.0
+pip>=9.0.2
```

Comment:
Not really related to this PR, but is this line still relevant now that we require Pytrhon 3.7+?

Reply:
I don't think so, but technically you don't have to have pip installed in an environment either, so I don't think it's hurting anything by being explicit.